src/js/utils/apiClient.jsx

```diff
@@ -97,6 +97,9 @@ export const handleError = (error) => {
       });
       break;
     default:
+      if (error?.code === AxiosError.ERR_NETWORK) {
+        break;
```

Comment:
Add a comment why it's done like that.

Reply:
@kchelstowski How about:
```
// We don't want to spam "network errors" popups
// when a user loses the connection, because 
// we are using the "Lost connection" message 
// for this, plus we are going to add a visual
// indicator for this <put a number of the ticket you created here>
```